depobs/website/models.py

```diff
@@ -288,6 +302,12 @@ def get_vulnerability_counts(package: str, version: str):
     ).filter(Advisory.package_name==PackageVersion.name
     ).group_by(Advisory.package_name, PackageVersion.version, Advisory.severity)
 
+def get_vulnerabilities(package: str, version: str):
+    return db_session.query(Advisory.package_name, PackageVersion.version, Advisory.severity, Advisory.url, Advisory.title
+    ).filter_by(package_name=package
+    ).filter(PackageVersion.version==version
+    ).filter(Advisory.package_name==PackageVersion.name)
```

Comment:
not sure if we need an `.all()` or if Flask will consume the query and yield it as an iterator. Probably not worth worrying about until we find a package with a ton of vulnerabilities.